packages/ember-htmlbars/tests/helpers/component_test.js

```diff
@@ -1,398 +0,0 @@
-  view = EmberView.create({
-    [OWNER]: owner,
-    location: 'Caracas',
-    template: compile('{{#component "does-not-exist" location=view.location}}arepas!{{/component}}')
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /HTMLBars error: Could not find component named "does-not-exist"./);
-});
-
-QUnit.test('component helper properly invalidates hash params inside an {{each}} invocation #11044', function() {
-  owner.register('component:foo-bar', Component.extend({
-    willRender() {
-      // store internally available name to ensure that the name available in `this.attrs.name`
-      // matches the template lookup name
-      set(this, 'internalName', this.attrs.name);
-    }
-  }));
-  owner.register('template:components/foo-bar', compile('{{internalName}} - {{attrs.name}}|'));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    items: [
-      { name: 'Robert' },
-      { name: 'Jacquie' }
-    ],
-    template: compile('{{#each view.items as |item|}}{{component "foo-bar" name=item.name}}{{/each}}')
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Robert - Robert|Jacquie - Jacquie|', 'component was rendered');
-
-  run(function() {
-    set(view, 'items', [
-      { name: 'Max' },
-      { name: 'James' }
-    ]);
-  });
-  equal(view.$().text(), 'Max - Max|James - James|', 'component was updated and re-rendered');
-});
-
-QUnit.test('dashless components should not be found', function() {
-  expect(1);
-
-  owner.register('template:components/dashless', compile('Do not render me!'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{component "dashless"}}')
-  }).create();
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /You cannot use 'dashless' as a component name. Component names must contain a hyphen./);
-});
-
-QUnit.test('positional parameters does not clash when rendering different components', function(assert) {
-  owner.register('component:normal-message', Component.extend({
-    something: null
-  }).reopenClass({
-    positionalParams: ['something']
-  }));
-
-  owner.register('component:alternative-message', Component.extend({
-    something: null
-  }).reopenClass({
-    positionalParams: ['something']
-  }));
-
-  owner.register('template:components/normal-message', compile('Say: {{something}}!'));
-  owner.register('template:components/alternative-message', compile('---: {{something}}!'));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    messageType: 'normal-message',
-    message: 'Hello',
-    template: compile('{{component view.messageType view.message}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'Say: Hello!');
-
-  run(function() {
-    set(view, 'messageType', 'alternative-message');
-  });
-
-  equal(view.$().text(), '---: Hello!');
-});
-
-QUnit.test('positional parameters does not pollute the attributes when changing components', function(assert) {
```

Comment:
moved to `'@htmlbars positional parameters does not pollute the attributes when changing components`